lib/optimist.rb

```diff
@@ -996,6 +1015,20 @@ def multi_arg? ; true ; end
 ##   ## if called with --monkey
 ##   p opts # => {:monkey=>true, :name=>nil, :num_limbs=>4, :help=>false, :monkey_given=>true}
 ##
+## Settings:
+##   Optimist::options and Optimist::Parser.new accept +settings+ to control how
+##   options are interpreted.  These settings are given as hash arguments, e.g:
+##
+##   opts = Optimist::options(ARGV, exact_match: false) do
+##     opt :foobar, 'messed up'
+##     opt :forget, 'forget it'
+##   end
+##
+##  +settings+ include:
+##  * :exact_match  : (default=false) Allow minimum unambigous number of characters to match a long option
```

Comment:
```suggestion
##  * :exact_match  : (default=true) Allow minimum unambigous number of characters to match a long option
```

Reply:
Switching the default to true, I think changes how we should describe the option...maybe something like:

```suggestion
##  * :exact_match  : (default=true) When set to false, allow minimum unambigous number of characters to match a long option.
```